bundle/regal/rules/bugs/inconsistent_args.rego

```diff
@@ -0,0 +1,63 @@
+# METADATA
+# description: Inconsistently named function arguments
+package regal.rules.bugs["inconsistent-args"]
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.regal.ast
+import data.regal.result
+
+report contains violation if {
+	count(ast.functions) > 0
+
+	# Comprehension indexing — as made obvious here it would be great
+	# to have block level scoped ignore directives...
+	function_args_by_name := {name: args_list |
+		some i
+
+		# regal ignore:prefer-some-in-iteration
+		name := ast.ref_to_string(ast.functions[i].head.ref)
+		args_list := [args |
+			some j
+
+			# regal ignore:prefer-some-in-iteration
+			ast.ref_to_string(ast.functions[j].head.ref) == name
+
+			# regal ignore:prefer-some-in-iteration
+			args := ast.functions[j].head.args
+		]
+		count(args_list) > 1
```

Comment:
Won't the arg list size always be > 1, as each rule is counted at least twice, once when iterating `i` and once for `j`?

Reply:
But the args_list only iterates on `j`.